src/templates/dso_api/dynamic_api/mvt_index.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>{% load i18n %}
+<html>
+<head>
+  <title>MVT datasets</title>
+  {% block extrahead %}{% endblock %}
+</head>
+<body>
+  <main>
+    <h1>Index of datasets supporting Mapbox Vector Tiles (MVT)</h1>
+
+    <ul>
+    {% for name, fields, schema in datasets %}
+      <li>
+        <span>{% if schema.title %}{{ schema.title }}{% else %}{{ name|title }}{% endif %}:</span>
+        <ul>
+        {% for field in fields %}
+          <li>
+            <tt><script language="javascript">document.write(location.origin);</script>/v1/mvt/{{ name }}/{{ field }}/{z}/{x}/{y}.pbf</tt>
```

Comment:
zonde om hier javascript te gebruiken. De server weet zijn origin ook :-)

`request.build_absolute_uri()` werkt op basis van de `HTTP_HOST` e.d. parameters.
Je kan zelfs kiezen voor `request.build_absolute_uri(reverse("dynamic_api:mvt_index"))` als je het URL pattern voor de index ook een naam geeft. Met `reverse()` kan je in Django de URL resolven zoals je die in `urls.py` geconfigureerd hebt, en bij reorganisatie gaat dan alles vanzelf mee.